source/features/index.tsx

```diff
@@ -194,7 +194,7 @@ function enforceDefaults(
 const add = async (id: FeatureID, ...loaders: FeatureLoader[]): Promise<void> => {
 	/* Feature filtering and running */
 	const options = await globalReady;
-	if (options[`feature:${id}`] === false) {
+	if (id !== __filebasename && !options[`feature:${id as unknown as string}`]) {
```

Comment:
This wasn't necessary before because the `FeatureConfig` type was `string | boolean` due to some TypeScript restrictions. The current code allows for correct types so XO correctly complained about this.

However `index` now needs its own check because `options.['feature:index']` isn't `false`, but still falsy.